src/http/local_async_normal.hpp

```diff
@@ -41,8 +48,10 @@ template<>
 struct http_async_connection<Tag, version_major, version_minor>
     : async_connection_base<Tag, version_major, version_minor>,
       protected http_async_protocol_handler<Tag, version_major, version_minor>,
-      boost::enable_shared_from_this<
+      std::enable_shared_from_this<
           http_async_connection<Tag, version_major, version_minor> > {
+  http_async_connection(http_async_connection const&) = delete;
```

Comment:
Why is this being set to `delete`?

Reply:
This file is effectively a copy of [boost/network/protocol/http/client/connection/async_normal.hpp](https://github.com/cpp-netlib/cpp-netlib/blob/e80404ec3222b0bd516ea4ad475c66c7222a4350/boost/network/protocol/http/client/connection/async_normal.hpp) -- I just pulled in the changes from the original. Let me separate such changes out into a separate commit, for ease of reviewing. Stay tuned.